<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeiboSpidersClass</class>
 <widget class="QWidget" name="WeiboSpidersClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WeiboSpiders</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGroupBox" name="focuUesrBox">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>201</width>
     <height>381</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
font-size: 16px;
background-color: rgb(225, 225, 225);
border-radius: 10px;
}
</string>
   </property>
   <property name="title">
    <string>当前已添加用户</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>181</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListView{
font-size: 14px;
border: 5px solid white;
border-radius: 10px;
}

QListView::item:hover{
background-color: transparent;
padding: 5px;
border-left: 3px solid rgb(130, 130, 130);
}

QListView::item:selected{
background-color: transparent;
color: black;
padding: 10px;
border-left: 3px solid black;
}

QScrollBar::vertical{
background-color: rgb(0, 255, 0);
width: 5px;
}

</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="addUidButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>340</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #ffffff;
	font-size: 20px;
	border: 1px solid #dcdfe6;
	padding: 5px;
	border-radius: 8px;
}

QPushButton:hover {
    background-color: #ecf5ff;
    color: #409eff;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="uidLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-color: rgb(255, 255, 255);
	border: 1px solid white;
	border-radius: 10px;
	padding: 5px;
}</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>请输入微博用户uid</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>400</y>
     <width>141</width>
     <height>38</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>20</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #ffffff;
    border: 1px solid #dcdfe6;
    padding: 10px;
    border-radius: 5px;
}

QPushButton:hover {
    background-color: #ecf5ff;
    color: #409eff;
}

QPushButton:pressed, QPushButton:checked {
    border: 1px solid #3a8ee6;
    color: #409eff;
}

#button3 {
    border-radius: 20px;
}</string>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="fnBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>511</width>
     <height>431</height>
    </rect>
   </property>
   <property name="title">
    <string>微博定时工具</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>400</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #ffffff;
    border: 1px solid #dcdfe6;
    padding: 10px;
    border-radius: 5px;
}
QPushButton:hover {
    background-color: #ecf5ff;
    color: #409eff;
}</string>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="WeiboSpiders.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>WeiboSpidersClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>675</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
